### PR TITLE
Update kubevirtci.

### PR DESCRIPTION
**Pull request by maya-r**
Changes default KUBEVIRT_PROVIDER to k8s-1.18 that works, the previous
CDI-specific change was wrong and got overridden by another
'make generate'.

Removes ocp and okd kubevirtci providers.

```release-note
NONE
```

### CODE REVIEW

File: cluster-sync/okd-4.3/hpp-master-mc.yaml [hunk 1]
```diff
@@ -1,27 +0,0 @@
-apiVersion: machineconfiguration.openshift.io/v1
```
Comment (awels) [inline cluster-sync/okd-4.3/hpp-master-mc.yaml:1]: Can you open a PR against maybe the hostpath-provisioner-operator with the contents of these two files (hpp-master-mc.yaml and hpp-worker-mc.yaml). There have been quite a few times I have pointed people to these files with regards to Open Shift.

Comment (awels) [reply cluster-sync/okd-4.3/hpp-master-mc.yaml:1]: Maybe add a contrib or something directory.

Comment (maya-r) [reply cluster-sync/okd-4.3/hpp-master-mc.yaml:1]: Here: https://github.com/kubevirt/hostpath-provisioner-operator/pull/73

Comment (awels) [reply cluster-sync/okd-4.3/hpp-master-mc.yaml:1]: thanks

**Pull request merged**